code/style.css

```
@@ -1,5 +1,270 @@
-/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+html {
+  height: 100%;
+  width: full-width;
+}
 
+head {
+  padding: 30px;
+  margin-bottom: 50px;
+  margin-top: 5px;
+  margin-left: 5px;
+  margin-right: 5px;
+}
 body {
-  background-color: lightblue;
+  height: 100%;
+  font-family: "Montserrat", sans-serif;
+  background-color: white;
+  margin-top: 95px;
+}
+
+h1 {
+  font-weight: bold;
+}
+
+h2 {
+  font-weight: bold;
+}
+
+.logo {
+  float: right;
+}
+.container2 {
+  position: absolute;
+  top: 0;
+  clear: none;
+  overflow: hidden;
+  background-color: white;
+}
+nav {
+  float: right;
+  position: absolute;
+  top: 0;
+  right: 30%;
+  padding: 30px;
+  overflow: hidden;
+  background-color: white;
+  height: 80px;
+  word-spacing: 5px;
+  margin-left: 50px;
+  margin-bottom: 50px;
+  text-transform: uppercase;
+}
+nav a {
+  float: left;
+  color: black;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+  word-spacing: 5px;
+}
+nav a:hover {
+  background-color: #ddd;
+  color: black;
+  border: black;
+  border-radius: 50px;
+}
+
+.appointment-button {
+  background-color: rgba(0, 128, 0, 0.312);
+  color: black;
+  margin: 5px;
+  padding: 10px;
+  border-radius: 50px;
+  word-spacing: 5px;
+  font-weight: 900;
+}
+
+a {
+  color: black;
+  font-style: normal;
+  font-weight: bold;
+  text-decoration: none;
+}
+
+.hero-image {
+  background-image: url("./pexels-karolina-grabowska-4239125.jpg");
+  height: 400px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  top: 50px;
+  margin-top: 50px;
+  margin-bottom: 10px;
+  width: full-width;
+}
+
+.hero-text {
+  text-align: center;
+  position: absolute;
+  top: 25%;
+  left: 5%;
+  color: white;
+  font-weight: bold;
+}
+
+button {
+  background-color: rgba(0, 128, 0, 0.312);
+  border-color: rgba(0, 128, 0, 0.312);
+  margin-top: 5px;
+  padding: 10px;
+  border-radius: 5px;
+  word-spacing: 5px;
+  font-weight: bold;
+  color: white;
+}
+button:hover {
+  background-color: gray;
+  border-color: grey;
+  margin-top: 5px;
+  padding: 10px;
+  border-radius: 5px;
+  word-spacing: 5px;
+  font-weight: bold;
+  color: white;
+}
+
+.container {
+  margin-top: 50px;
+  position: relative;
+  width: 33.33%;
+  height: 100%;
+  display: inline-flex;
+  flex-direction: row;
+  justify-content: center;
+  float: left;
+}
```

If you add a `.services` class to the parent div of the containers, it would simplify your css.
```suggestion
.services {
  display: flex;
  margin-top: 50px;
}

.container {
  position: relative;
  width: 33.33%;
  height: 100%;
  justify-content: center;
}
```